homeassistant/components/climate/nest.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, structure, device, temp_unit):
         self._unit = temp_unit
         self.structure = structure
         self.device = device
+        self._fan_list = ["On", "Auto"]
```

Comment:
import STATE_AUTO from climate, and define STATE_ON as on.
The use constants inside fan_list.